src/models/taskModel.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const taskSchema = new mongoose.Schema({
+    title: {
+        type: 'String',
+        required: [true, 'Please provide task title'],
+        description: {
+            type: 'String',
```

Comment:
the `desccription` seems to be under `title`, is this intentional?

Reply:
No, this was inadvertent, will correct